packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/queries.ts

```diff
@@ -0,0 +1,73 @@
+/* Total and circulating total supply
+
+https://github.com/input-output-hk/cardano-graphql/blob/610424b95053078c20be121d823ef0dd035711f0
```

Comment:
```suggestion
REVIEW: https://github.com/input-output-hk/cardano-graphql/blob/610424b95053078c20be121d823ef0dd035711f0
```
This has been a pattern we've used to add context for the purpose of review. Once we have verified this logic, the current context implementation becomes our reference, so we can remove this prior to merge.

Reply:
d7d100347e2fd124277f1d49bbd393cf2e4bba89